src/axeReporterEarl.ts

```diff
@@ -0,0 +1,22 @@
+import { RawResult, EarlResult } from './types'
+import * as context from './context.json'
+import axeResultToAssertion from './axeResultToEarl'
+
+export default function axeEarlReporter(
```

Comment:
This is a very strange function. Why not just call `createEarlReport` directly?

Reply:
Because of the callback.